webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -14,8 +13,13 @@ import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem
 import { TranslatedPlural } from '../../../Table/components/TranslatedPlural';
 import { hostIsNotRegistered } from '../hostDetailsHelpers';
 
-const HostDisks = ({ totalDisks }) => {
-  if (!totalDisks) return null;
+const HostDisks = ({ blockDevices }) => {
```

Comment:
I still think it's poor practice to apply this logic in the frontend. We've settled on the reported data facet to normalize all fact uploads. Ideally it would be implemented in https://github.com/theforeman/foreman/blob/develop/app/services/katello/rhsm_fact_parser.rb